bayes/src/main/java/com/enigmastation/ml/bayes/impl/SimpleClassifierImpl.java

```diff
@@ -222,7 +222,7 @@ private int totalCount() {
         return sum;
     }
 
-    protected Set<Object> categories() {
+    protected Set<Object> getCategoriesKeySet() {
```

Comment:
Should be getCategories(), Java is strongly typed. In Node I'd be fully on board. In Java, not so much.